Defs/ThingDefs/hayfeedingtrough.xml

```diff
@@ -2,12 +2,12 @@
 <Buildings>
 
   <ThingDef ParentName="FurnitureBase">
-    <defName>heyfeedingtrough</defName>
```

Comment:
I don't know if this breaks compatibility; I suspect it probably does as the existing name will appear in save files, so it might have to stay the same unless rimworld provides a mechanism for migrating save data. I'm not familiar with the mod API